test/tus/tus_integration_test.go

```
@@ -996,3 +997,107 @@ func TestTusAbortedPatchKeepsStoredChunks(t *testing.T) {
+	require.NoError(t, err)
+	createReq.Header.Set("Tus-Resumable", TusVersion)
+	createReq.Header.Set("Upload-Length", strconv.Itoa(len(testData)))
+
+	createResp, err := client.Do(createReq)
+	require.NoError(t, err)
+	createResp.Body.Close()
+	require.Equal(t, http.StatusCreated, createResp.StatusCode)
+	uploadLocation := createResp.Header.Get("Location")
+
+	// PATCH A promises one sub-chunk and stalls mid-body, holding the session.
+	conn, err := net.Dial("tcp", "127.0.0.1:"+testFilerPort)
```

golangci-lint (2.12.2)

[error] net.Dial must not be called. use (*net.Dialer).DialContext

(noctx)